### ISSUE TITLE
Thickness and Alpha are missing in Glow

### ISSUE THREAD

**Issue by nikita-tarasov3** [danielkrupinski/Osiris#1928]
Thickness and Alpha are missing in Glow. What happened with them? @danielkrupinski

**Comment by danielkrupinski** [comment]
Thickness is unneeded, alpha can be set in color picker.

**Comment by adampala123** [comment]
I personally really liked thickness. Hate having those fat default glow options.

**Comment by nikita-tarasov3** [comment]
> I personally really liked thickness. Hate having those fat default glow options.

same

**Comment by RyDeem** [comment]
i agree with Adampala123, it was not usefull, did not do anything for most of the glows, but for some it made them look p

Btw @danielkrupinski will you merge the old ESP features like viewtracer(where other player is looking) back in to stemproof esp?
Because now it seems you just took one system and replaced it with the other one. so we lost the features we hjad and gained new different ones. i like streamproof but not so much to lose some old and loved features...

Basically my point is - upgrade dont sidegrade.

**Comment by danielkrupinski** [comment]
"Fat default glow" is gone now.
